src/share.js

```diff
@@ -90,6 +92,11 @@ class Share extends BasePlugin {
     }
   }
 
+  _getVideoDesc(): string {
+    let name = coreUtils.Object.getPropertyPath(this.player.config, 'sources.metadata.name') || 'the video';
```

Comment:
should "the video" be translated ?

Reply:
I just copied the function as is, it already was with "the video" hardcoded. i need to change it?